compiler/next/test/resolution/testResolve.cpp

```diff
@@ -128,11 +130,11 @@ static void test2() {
     context->collectGarbage();
   }
 }
-
+*/
 
 int main() {
-  test1();
-  test2();
+  //test1();
```

Comment:
Yes, I have removed this resolution code because I'm rewriting it to be more complete, cover more cases, and to use the scope resolution work in this PR. However since I'm going to be rewriting it in a next PR I left it commented out rather than deleting it all. I could delete it if that's important to you.